index.js

```diff
@@ -2,63 +2,112 @@
 'use strict';
 
 var path = require('path');
+var fs = require('fs-extra');
+var existsSync = require('exists-sync');
 
 var EventEmitter = require('events').EventEmitter;
 var mergeTrees = require('broccoli-merge-trees');
 var VersionChecker = require('ember-cli-version-checker');
 
-var patchEmberApp     = require('./lib/ext/patch-ember-app');
 var fastbootAppModule = require('./lib/utilities/fastboot-app-module');
 
 var filterInitializers = require('fastboot-filter-initializers');
```

Comment:
Nope, this was the dirty ember-app private patching that we was causing issues and perf problems.